csharp/Microsoft.Azure.Databricks.Client/CronSchedule.cs

```diff
@@ -15,5 +15,27 @@ public class CronSchedule
         /// </summary>
         [JsonProperty(PropertyName = "timezone_id")]
         public string TimezoneId { get; set; }
+
+        /// <summary>
+        /// Indicate whether this schedule is paused or not.
+        /// </summary>
+        [JsonProperty(PropertyName = "pause_status")]
```

Comment:
I think you also need a [JsonConverter(typeof(StringEnumConverter))] attribute here.

Reply:
Done!